src/components/Input/Inputwithlabel.component.jsx

```diff
@@ -2,12 +2,12 @@ import "./inputwithlabel.style.css";
 function InputWithLabel({ Label, placeholder, multiline }) {
     return (
         <div className="input">
-            <label className="label">{Label}</label>
+            <label className="label" htmlFor="labelId">{Label}</label>
             {
                 multiline ? (
-                    <textarea placeholder={placeholder} />
+                    <textarea className="inputBar" placeholder={placeholder} />
                 ):(
-                    <input type="text" placeholder={placeholder}/>
+                    <input className = "inputBar" type="text" id="labelId" placeholder={placeholder}/>
```

Comment:
the `htmlFor` and `id` attributes have static values, they should take their value from a prop or just use the `Label` prop as the `id`.
side note for consistency: `Label` would be better if it was all lower case because upper case first letter is usually used for classes.